### DISCUSSION TITLE
Voice Control Power On

### DISCUSSION THREAD

**Discussion by DVGPR0** [gopro/labs#353] [Feature Requests]
Not sure if this is a hardware thing or not but being able to turn on the cam with my voice would be really great as most of the time I'm using my camera, I'm on a bike and can't use my hands without stopping.

**Comment by dnewman-gpsw** [comment]
This not a Labs feature, but the camera does have voice commands.  Check those out.

**Reply by DVGPR0** [reply]
Yeah, I know. I was hoping Labs might be able to add additional voice commands that don't come stock. I love the current build, BTW.

**Reply by dnewman-gpsw** [reply]
I asked how to add commands, and it is not cheap hack.  It involves a look of language training, so outside of what Labs can do.
